suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheet.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles/src';
+import { Icon } from '@trezor/icons/src';
+import { WalletAccountTransaction } from '@suite-common/wallet-types/src';
+import { Box, Card, Text, VStack } from '@suite-native/atoms/src';
+import { useFormatters } from '@suite-common/formatters/src';
```

Comment:
remove src from imports. Only import with @trezor/styles instead of @trezor/styles/src